contrib/opencensus-ext-azure/opencensus/ext/azure/metrics_exporter/statsbeat_metrics/__init__.py

```diff
@@ -15,41 +15,45 @@
 import threading
 
 from opencensus.ext.azure.metrics_exporter import MetricsExporter
-from opencensus.ext.azure.metrics_exporter.heartbeat_metrics.heartbeat import (
-    HeartbeatMetric,
+from opencensus.ext.azure.metrics_exporter.statsbeat_metrics.statsbeat import (
```

Comment:
Statsbeat is not a replacement of heartbeat.  Might want to keep Hearbeat metrics. they're very diff.

Reply:
i realized statsbeat_metrics.statsbeat is not a package name.. rather the class name.

Reply:
https://github.com/census-instrumentation/opencensus-python/pull/1053#discussion_r678619901